RealmSwift-swift1.2/RealmConfiguration.swift

```diff
@@ -147,10 +147,12 @@ extension Realm {
 
         internal var rlmConfiguration: RLMRealmConfiguration {
             let configuration = RLMRealmConfiguration()
-            if self.path != nil {
+            if path != nil {
```

Comment:
Why remove the `self.` when most other lines in the function use it, including the next line which assigns to `configuration.path`?